integration/modeshape-jbossas-integration-tests/src/main/webapp/WEB-INF/jboss-deployment-structure.xml

```diff
@@ -0,0 +1,10 @@
+<jboss-deployment-structure>
```

Comment:
why is this explicit dependency file required by the test ? 
The WF config specifies a module dependency in the actual index-provider element:
```
<index-provider name="elasticsearch" classname="org.modeshape.jcr.index.elasticsearch.EsIndexProvider" module="org.modeshape.index-provider.elasticsearch"/>

as such, the CP should be resolved at runtime correctly.